src/iscreen/iscreen.h

```diff
@@ -33,7 +33,7 @@ const int  iS_STR_SEG_START_COL	= 144;
 //const int  iS_STR_SEG_START_COL	  = 80;
 
 const int  iSTR_LEN			= 50;
-const int  iS_STR_LEN		= 100;
+const int  iS_STR_LEN		= 512;

```

Comment:
Why you changed this length?

Reply:
That.s the reason of the overflows, the length of data read were bigger than this.